Makefile

```diff
@@ -1,18 +1,69 @@
 ARCHS=linux linux64 mac64 win32
-MANIFEST=manifest.json
 
-VERSION=`grep -Eo '"version"\:\s*"[^"]+' $(MANIFEST) | grep -Eo '[0-9].*'`
+EXTENSION_NAME=adb-extension
+VERSION=0.0.2
```

Comment:
Should the version have "pre" suffix here?

Reply:
Yes good point I should modify that here and then do a first release PR.